sdk/core/System.ClientModel/src/Internal/CancellationHelper.cs

```diff
@@ -66,7 +67,8 @@ internal static void ThrowIfCancellationRequestedOrTimeout(CancellationToken mes
             throw CreateOperationCanceledException(
                 innerException,
                 timeoutToken,
-                $"The operation was cancelled because it exceeded the configured timeout of {timeout:g}. ");
+                $"The operation was cancelled because it exceeded the configured timeout of {timeout:g}. " +
+                    $"The default network timeout can be adjusted in the client options type via {nameof(ClientPipelineOptions)}.{nameof(ClientPipelineOptions.NetworkTimeout)}.");
```

Comment:
Could we also set the Exception.HelpLink to an md file showing how to do it?

Reply:
Also, maybe we can simplify to: "The timeout can be adjusted by passing a custom {nameof(ClientPipelineOptions)}.{nameof(ClientPipelineOptions.NetworkTimeout) value to the client's constructor"

Reply:
Using the helpLink is not something we usually do. If we include help links it is in the message directly.